clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -38,8 +38,13 @@ include "mlir/IR/CommonAttrConstraints.td"
 // CIR Ops
 //===----------------------------------------------------------------------===//
 
+class LoweringInfo {
```

Comment:
Add a comment here explaining that this is used to communicate to tablegen some direct lowering approach and perhaps an example of how it can be used.

Reply:
Updated.